hknweb/candidate/templates/candidate/index.html

```diff
@@ -13,242 +13,148 @@
+                    {% endif %}
+                    {{ interactivities.officer_hangout.title }}
+                </li>
+                <ul class="conf-events">
+                    {% for event in confirmed_events.hangout %}
+                        <li>{{ event.name }}<br/></li>
+                    {% endfor %}
                 </ul>
-
-                <h4>Bit-Byte</h4>
-                <ul class="checkboxes">
-                    <li>
-                        <!-- do not distinguish between boba and adventures-->
-                        {% if req_statuses.bitbyte %}
-                            <span class="req done"></span>
-                        {% else %}
-                            <span class="req not-done"></span>
-                        {% endif %}
-                        {{ num_bitbytes }} Bit-Byte Activities Completed
-                    </li>
-                    <a href="{% url 'candidate:bitbyte' %}">
-                        (View/Request Bit-Byte Confirmations)</a>
+                <ul class="unconf-events">
+                    {% for event in unconfirmed_events.hangout %}
+                        <li>{{ event.name }}<br/></li>
+                    {% endfor %}
                 </ul>
-            </div>
-        </section>
-        <br>
-        <section>
-            <h2>Upcoming Events</h2>
-            {% if upcoming_events %}
-            <table class="full-table">
-                <thead>
-                    <th align="left">Event</th>
-                    <th align="left">Time</th>
-                    <th align="left">Location</th>
-                </thead>
+            </ul>
 
-                <tbody>
-                {% for event in upcoming_events %}
-                    <tr>
-                        <td><a href="{% url 'events:detail' event.id %}"> {{ event.name }} </a></td>
-                        <td>{{event.start_time|date:"D m/d h:i A"}} - {{ event.end_time|date:"h:i A"}}</td>
-                        <td>{{ event.location }}</td>
-                    </tr>
-                {% endfor %}
-                </tbody>
-            </table>
-            {% endif %}
-        </section>
-    </div>
+            <h4>{{ bitbyte.title }}</h4>
+            <ul class="checkboxes">
+                <li>
+                    <!-- do not distinguish between boba and adventures-->
+                    {% if bitbyte.status %}
+                        <span class="req done"></span>
+                    {% else %}
+                        <span class="req not-done"></span>
+                    {% endif %}
+                    {{ bitbyte.num_bitbytes }} Bit-Byte Activities Completed
+                </li>
+                <a href="{% url 'candidate:bitbyte' %}">
+                    (View/Request Bit-Byte Confirmations)</a>
+            </ul>
+
+            <h4>Miscellaneous Requirements</h4>
+            <ul class="events-list" type="1">
+                <li>Administer 1-2 Course Surveys</li>
```

Comment:
I think we can remove the course survey requirement for Fall 2020

Reply:
Yeet!

Reply:
![compserv candidate portal without course surveys req](https://user-images.githubusercontent.com/46059916/95279667-2052f400-0808-11eb-9de6-62dfedcae877.PNG)